tableauserverclient/server/endpoint/groups_endpoint.py

```diff
@@ -59,10 +73,6 @@ def remove_user(self, group_item, user_id):
         self._remove_user(group_item, user_id)
         try:
             users = group_item.users
```

Comment:
This is actually unnecessary because every call is a new Pager now, but we could return the list from `add_user` or we can just remove this from the call enitrely